UnitedIncome/components
# CategoryBubbleSelector -> TypeScript -> RTL

### Description
> Please provide a brief description of your changes below.

Converts the `CategoryBubbleSelector` to use TypeScript and React Testing Library.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Tests should run.
* Also used `memo`.
